services/credential-issuance-server/src/agent.ts

```diff
@@ -38,13 +38,14 @@ class Agent {
     return this.signifyApi.resolveOobi(url);
   }
 
-  async issueAcdcCredentialByAid(schemaSaid, aid, name?) {
+  async issueAcdcCredentialByAid(schemaSaid, aid, name?, LEI?) {
```

Comment:
Can we just have an `attributes` field? And this gets directly passed through? Then it's up to the caller to provide the correct attributes.

(perhaps you can test what happens if we produce the wrong ones, and catch any exception if there is one and return 400?)